DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx

```diff
@@ -266,6 +266,17 @@ void DataRequest::requestMCHClusters(bool mc)
   requestMap["clusMCH"] = mc;
 }
 
+void DataRequest::requestHMPClusters(bool mc)
+{
+  addInput({"hmpidcluster", "HMP", "CLUSTERS", 0, Lifetime::Timeframe});
+  addInput({"hmpidtriggers", "HMP", "CLUSREFS", 0, Lifetime::Timeframe});
```

Comment:
The special vector of charges is part of the clusters info, should be added here